From e99cb3ecd8334ca21e01ff9a79a916693f58f1fb Mon Sep 17 00:00:00 2001
From: Randy Dunlap <rdunlap@infradead.org>
Date: Tue, 21 Jul 2026 10:21:32 -0700
Subject: [PATCH 08804/15333] nvdimm-btt: clean up kernel-doc warnings

Add one missing kernel-doc description and drop one that shouldn't be
here in order to avoid kernel-doc warnings:

Warning: ./drivers/nvdimm/btt.h:232 struct member 'nd_region' not described in 'btt'
Warning: ./drivers/nvdimm/btt.h:232 Excess struct member 'lanes' description in 'btt'

Signed-off-by: Randy Dunlap <rdunlap@infradead.org>
Reviewed-by: Dave Jiang <dave.jiang@intel.com>
Link: https://patch.msgid.link/20260721172132.3718892-2-rdunlap@infradead.org
Signed-off-by: Alison Schofield <alison.schofield@intel.com>
---
 drivers/nvdimm/btt.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/nvdimm/btt.h b/drivers/nvdimm/btt.h
index 0c76c0333f6e..1ed245c18250 100644
--- a/drivers/nvdimm/btt.h
+++ b/drivers/nvdimm/btt.h
@@ -210,7 +210,7 @@ struct badblocks;
  * @lbasize:		LBA size as requested and presented to upper layers.
  *			This is sector_size + size of any metadata.
  * @sector_size:	The Linux sector size - 512 or 4096
- * @lanes:		Per-lane spinlocks
+ * @nd_region:		&struct nd_region pointer
  * @init_lock:		Mutex used for the BTT initialization
  * @init_state:		Flag describing the initialization state for the BTT
  * @num_arenas:		Number of arenas in the BTT instance
-- 
2.55.0

